packages/docs/src/locales/en/en.json

```diff
@@ -1151,63 +1151,63 @@
-    "description": "This page is intended for vuestic-ui contributors: it will explain how to create and modify documentation.",
+    "description": "This page is intended for Vuestic-UI contributors. It explains the ways to create and modify the documentation.",
     "introduction": {
       "title": "Introduction",
-      "description": "Instead of using the established documentation system, such as [vue-press](https://vuepress.vuejs.org/)[[target=_blank]] or [docsify](https://docsify.js.org/#/)[[target=_blank]], we’ve decided to create a system specifically tailored for Vuestic. It’s meant to provide excellent flexibility for future growth."
+      "description": "Instead of using the established documentation system, such as [vue-press](https://vuepress.vuejs.org/)[[target=_blank]] or [docsify](https://docsify.js.org/#/)[[target=_blank]], we decided to build a system specifically tailored for Vuestic. It’s meant to provide an excellent flexibility for the future growth."
     },
     "pageConfig": {
       "title": "Page Config",
-      "generate": "If you’re trying to document a component from scratch, use the `yarn generate:component` command, which will guide you through the process.",
-      "description": "Documentation has a flat structure - list of blocks. Markdown is supported in all text blocks. Here are the blocks you can use:"
+      "generate": "If your goal is to document a component from scratch, use the `yarn generate:component` command, which will guide you through the process.",
+      "description": "Translation strings are structured in a flat manner — just a list of blocks. Markdown is supported for all the text blocks. Here is the list of blocks you can use:"
     },
     "blocktypes": {
       "title": {
         "title": "Title",
-        "text": "Page title is mandatory for documentation pages.",
+        "text": "Page title is **mandatory** for documentation pages.",
         "example": "Title (example)"
       },
       "subtitle": {
         "title": "Subtitle",
-        "text": "Used for: Examples, API, FAQ. Think `h2`.",
+        "text": "Used for examples, API, FAQ. Think `h2`.",
         "example": "Subtitle (example)"
       },
       "headline": {
         "title": "Headline",
-        "text": "The block is used to mark titles of examples or to FAQ questions. Think `h3`.",
+        "text": "The `headline` block is used to mark the titles of examples and the FAQs. Think `h3`.",
         "example": "Headline (example)"
       },
       "paragraph": {
         "title": "Paragraph",
-        "text": "Should be used for all non-special text blocks. For links to external resources, you can specify the **target** attribute in markdown markup as follows: `[name](href)[[target=_blank]]`.",
+        "text": "Should be used for all the regular text blocks. For links to external resources you can specify the **target** attribute in markdown markup as follows: `[name](href)[[target=_blank]]`.",
```

Comment:
@Derranion And I mean **the** regular text blocks (regular as a type - i.e. just a plain text).

Reply:
@smellyshovel Great, I meant cases like this:

"text": "The `headline` block is used to mark the titles of examples and the FAQs. Think `h3`.",
 
 "The titles" means one specific titles list. ( for example: "MyUniqueTitle, AnotherOne" )
 
 But it could be any list not only this one.

Reply:
Pretty delicate thing and depends heavily on a given context IMHO. In this very specific case I'm talking of **the titles** of examples since I'm treating **the title** as a **part of an example** (read "a pretty much specific building block of a given example"), but not as **a title** on its own (as some random text).

Not sure about these usages however, will leave it as they are for now, though I'm not against anybody editing this wording later.